ksqldb-streams/src/test/java/io/confluent/ksql/execution/streams/TableTableJoinBuilderTest.java

```diff
@@ -193,7 +193,12 @@ public void shouldReturnCorrectSchema() {
     // Then:
     assertThat(
         result.getSchema(),
-        is(JoinParamsFactory.create(R_KEY, LEFT_SCHEMA, RIGHT_SCHEMA).getSchema())
```

Comment:
Side cleanup: We should not use `JoinParamsFactory` to compute the expected result, because `JoinParamsFactory` is use to compute the actual `result` within `join.build(...)`. Otherwise we have a cyclic tes "dependency"...

Same below.